src/components/ScopeTree/ScopeTree.tsx

```diff
@@ -31,6 +31,22 @@ type ScopeTreeProps = {
   restrictToPractitionerPerimeter?: boolean
 }
 
+const orderRowsByAccess = (rowA: ScopeTreeRow, rowB: ScopeTreeRow): number => {
+  if (rowA.access === rowB.access) {
+    const nameA = rowA.name
+    const nameB = rowB.name
+
+    if (nameA < nameB) return -1
+    if (nameA > nameB) return 1
+  } else if (rowA.access) {
+    return -1
+  } else if (rowB.access) {
+    return 1
+  }
+
+  return 0
```

Comment:
I'd put the `return 0` at the end of `if (rowA.access === rowB.access)` and replace `else if (rowB.access)` by `else`